app/models/contribution.rb

```diff
@@ -6,6 +6,13 @@ class Contribution < ApplicationRecord
   include AASM
 
   enumerize :pr_state, :in => [:open, :closed, :merged], scope: :shallow, predicates: true
+  enumerize :rejected_reason, in: {
+    allocated_in_the_project: 0,
+    project_is_not_on_the_curated_list: 1,
```

Comment:
we don't need this, the bot doesn't fetch PRs of projects that are not in the curated list

Reply:
add a new instead, wrong understanding of issue

Reply:
No problem, although this reason was specified in the issue.

Added a `wrong_understading_of_issue` :+1: